python/pyproject.toml

```diff
@@ -16,4 +16,11 @@
 # under the License.
 
 [build-system]
-requires = ["setuptools", "wheel", "setuptools_scm", "cython >= 0.29"]
+requires = [
+    "cython >= 0.29",
+    "numpy==1.14.5; python_version<'3.7'",
+    "numpy==1.16.0; python_version>='3.7'",
```

Comment:
Is a precise version required? It would seem better to require a minimum version, if necessary (our use of Numpy C APIs is pretty basic, so I'm not sure it's needed).

Reply:
This is the minimal supported version we can build with. These are build-time only requirements. For NumPy, this means this is the lowest version that the resulting wheel is compatible with. Leaving this open would mean that the module would always be built uisng the latest numpy and then also only be usable with the latest numpy.

Reply:
I see, thank you.